src/CMakeLists.txt

```diff
@@ -12,4 +12,29 @@ project(vocoder VERSION 0.0)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)

```

Comment:
is this cmake file for tests only? I'd say it's poor form if you're forced to build the tests when you only want the executable.

Reply:
This is the main CMake file which builds the executable and tests, suppose it could probably be split up but would require some thought.

Reply:
I would assume you could do `make vocoder` to only build the project

Reply:
confirmed ^